src/content_script.js

```diff
@@ -296,29 +300,60 @@ function getOffsetsAndApplyClass(elemRect, bodyRect, target, htmlBadgeDiv) {
   }
 }
 
+function isVisible(target) {
+  const styleTransform = ( window.getComputedStyle(target, false).getPropertyValue("transform") === "matrix(1, 0, 0, 0, 0, 0)" );
+  const styleHidden = ( window.getComputedStyle(target, false).getPropertyValue("visibility") === "hidden" );
+  const styleDisplayNone = ( window.getComputedStyle(target, false).getPropertyValue("display") === "none" );
```

Comment:
Some duplication with `isDisplayNone` here - can we refactor it away?

Reply:
Ahh. Nuking isDisplayNone for this one.